artifacts/default-cloud-director-ccm-crs-airgap.yaml.template

```diff
@@ -193,6 +193,9 @@ spec:
         - name: vcloud-ccm-config-volume
           configMap:
             name: vcloud-ccm-configmap
+        - name: vcloud-capvcd-zones
```

Comment:
please ensure any reference in capvcd repo for cpi configmaps are being tracked

Reply:
the config map is not referred in CAPVCD repo as of now